3-semantic-html/basic.html

```diff
@@ -0,0 +1,46 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Semantic Exercise</title>
+</head>
+
+<body>
+
+    <header>
+        <!--Header content-->
+        <nav>
+            <!--Nav content-->
+        </nav>
+    </header>
+    <main>
+        <article>
+            <header>
+                <h1>PUSH-DEV SEMANTIC EXERCISE</h1>
```

Comment:
```suggestion
                <h1>Push-dev semantic exercise</h1>
```
In HTML is not very common to use all caps.